dev-python/doit-py/doit-py-0.5.0.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="doit tasks for python stuff"
+HOMEPAGE="https://pythonhosted.org/doit-py/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="
+	test? (
+		dev-python/pyflakes[${PYTHON_USEDEP}]
+		dev-python/coverage[${PYTHON_USEDEP}]
+		app-text/hunspell
+	)"
+RDEPEND="
+	dev-python/configclass[${PYTHON_USEDEP}]"
+
+distutils_enable_sphinx doc
```

Comment:
```
 * Using python3.8 in global scope
 * python3_8: running distutils-r1_run_phase python_compile_all
 * ERROR: dev-python/doit-py-0.5.0::testworld failed (compile phase):
 *   doc/conf.py not found, distutils_enable_sphinx call wrong
 * 
 * Call stack:
 *     ebuild.sh, line  125:  Called src_compile
 *   environment, line 2910:  Called distutils-r1_src_compile
 *   environment, line 1180:  Called _distutils-r1_run_common_phase 'python_compile_all'
 *   environment, line  427:  Called multibuild_foreach_variant 'distutils-r1_run_phase' 'python_compile_all'
 *   environment, line 2013:  Called _multibuild_run 'distutils-r1_run_phase' 'python_compile_all'
 *   environment, line 2011:  Called distutils-r1_run_phase 'python_compile_all'
 *   environment, line 1168:  Called python_compile_all
 *   environment, line 2268:  Called sphinx_compile_all
 *   environment, line 2894:  Called die
 * The specific snippet of code:
 *       [[ -f ${confpy} ]] || die "${confpy} not found, distutils_enable_sphinx call wrong";
 * 
 * If you need support, post the output of `emerge --info '=dev-python/doit-py-0.5.0::testworld'`,
 * the complete build log and the output of `emerge -pqv '=dev-python/doit-py-0.5.0::testworld'`.
 * The complete build log is located at '/var/tmp/portage/dev-python/doit-py-0.5.0/temp/build.log'.
 * The ebuild environment file is located at '/var/tmp/portage/dev-python/doit-py-0.5.0/temp/environment'.
 * Working directory: '/var/tmp/portage/dev-python/doit-py-0.5.0/work/doit-py-0.5.0'
 * S: '/var/tmp/portage/dev-python/doit-py-0.5.0/work/doit-py-0.5.0'

>>> Failed to emerge dev-python/doit-py-0.5.0, Log file:
```

Reply:
In my side compiles fine:
```
 * Using python3.8 in global scope
 * python3_8: running distutils-r1_run_phase python_compile_all
Running Sphinx v3.0.4
making output directory... done
WARNING: html_static_path entry '_static' does not exist
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 1 source files that are out of date
updating environment: [new config] 1 added, 0 changed, 0 removed
reading sources... [100%] index

looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] index

generating indices...  genindex py-modindexdone
highlighting module code... [ 20%] configclass
highlighting module code... [ 40%] doitpy.coverage
highlighting module code... [ 60%] doitpy.docs
highlighting module code... [ 80%] doitpy.package
highlighting module code... [100%] doitpy.pyflakes

writing additional pages...  searchdone
copying static files... ... done
copying extra files... done
dumping search index in English (code: en)... done
dumping object inventory... done
build succeeded, 1 warning.

The HTML pages are in doc/_build/html.
```
Can you please provide full build.log?

Reply:
```
>>> Downloading 'https://files.pythonhosted.org/packages/source/d/doit-py/doit-py-0.5.0.tar.gz'
--2020-07-09 07:24:58--  https://files.pythonhosted.org/packages/source/d/doit-py/doit-py-0.5.0.tar.gz
Resolving files.pythonhosted.org... 151.101.245.63, 2a04:4e42:3a::319
Connecting to files.pythonhosted.org|151.101.245.63|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://files.pythonhosted.org/packages/e5/a2/9311d9de7ec7ad771ab8b1ca17d139bfc745cb13f516b626fa38b1994bf5/doit-py-0.5.0.tar.gz [following]
--2020-07-09 07:24:58--  https://files.pythonhosted.org/packages/e5/a2/9311d9de7ec7ad771ab8b1ca17d139bfc745cb13f516b626fa38b1994bf5/doit-py-0.5.0.tar.gz
Reusing existing connection to files.pythonhosted.org:443.
HTTP request sent, awaiting response... 200 OK
Length: 6046 (5.9K) [application/x-tar]
Saving to: '/var/tmp/distfiles/doit-py-0.5.0.tar.gz.__download__'

     0K .....                                                 100% 6.79M=0.001s

2020-07-09 07:24:58 (6.79 MB/s) - '/var/tmp/distfiles/doit-py-0.5.0.tar.gz.__download__' saved [6046/6046]

!!! Fetched file: doit-py-0.5.0.tar.gz VERIFY FAILED!
!!! Reason: Filesize does not match recorded size
!!! Got:      6046
!!! Expected: 15174
Refetching... File renamed to '/var/tmp/distfiles/doit-py-0.5.0.tar.gz._checksum_failure_.ntnimagf'

!!! Couldn't download 'doit-py-0.5.0.tar.gz'. Aborting.
 * Fetch failed for 'dev-python/doit-py-0.5.0', Log file:
 *  '/var/tmp/portage/dev-python/doit-py-0.5.0/temp/build.log'
```

Oh yea I might've ran `repoman manifest` last time. The size it gets definitely implies a lot is missing from this package. Please clean your distfiles and try again?

This is on an empty, freshly updated stage3.

Reply:
OK, figured out what's problem. Indeed this is different tarball. I got it from Github, but forgot change SRC_URI. Now fixed.